src/microvis/core/_vis_model.py

```diff
@@ -177,21 +177,22 @@ def _on_any_event(self, info: EmissionInfo) -> None:
 # XXX: also ... this might make more sense as a method on the VisModel class
 # where we have access to the bound "T" type variable (could remove some casts)
 @lru_cache
-def validate_backend_class(
-    cls: type[VisModel], backend_class: Any
+def validate_adaptor_class(
+    model_class: Type[VisModel], adaptor_class: Any
```

Comment:
I noticed a commit where you put this as a method.  I think that's probably the right thing to do, if you want to redo it.  You can make it a `classmethod` and lose the cache for now (though I think we should try to retain caching: it could be as simple as a set of ids on the class for types that have already been validated)